maintainers/maintainer-list.nix

```diff
@@ -14967,4 +14967,10 @@
     github = "yisuidenghua";
     githubId = 102890144;
   };
+  mrtarantoga = {
```

Comment:
This list is sorted alphabetical.

Reply:
https://github.com/NixOS/nixpkgs/pull/186697/commits/60362606806856f21a3be0415620a8b7b136e6cf